templates/js/sso-tab-with-obo-flow/teamsapp.local.yml.tpl

```diff
@@ -44,7 +44,8 @@ provision:
   - uses: script
     with:
       run:
-        echo "::set-teamsfx-env TAB_DOMAIN=localhost";
+        echo "::set-teamsfx-env TAB_HOSTNAME=localhost";
+        echo "::set-teamsfx-env TAB_DOMAIN=localhost:53000";
```

Comment:
Do we still need variable of TAB_DOMAIN?

Reply:
Yes, we still need it, some AAD related configurations are still referencing it, for example:
```
"webApplicationInfo": {
        "id": "${{AAD_APP_CLIENT_ID}}",
        "resource": "api://${{TAB_DOMAIN}}/${{AAD_APP_CLIENT_ID}}"
    }
```